### PR TITLE
sahara: Add MapR and Storm to plugins enabled by default (fate#323275)

### PR DESCRIPTION
**Pull request by vuntz**
This is tech preview, so it doesn't hurt to expose more things by
default.

### CODE REVIEW

**Comment by toabctl** [comment]
@vuntz not sure but iirc there was a reason why theses are not enabled. maybe tempest will fail with theses plugins enabled...

**Comment by Itxaka** [comment]
I cant remember why I set only those plugins, @toabctl may be rigth on the tempest failures. I trigered a run with sahara enabled to see if it passes

**Comment by vuntz** [comment]
@Itxaka @toabctl mkcloud seems happy...

File: chef/data_bags/crowbar/template-sahara.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
       "cinder_instance": "none",
       "service_user": "sahara",
       "service_password": "",
-      "plugins": "vanilla,spark,cdh,ambari",
+      "plugins": "vanilla,spark,storm,cdh,ambari,mapr",
```
Comment (dirkmueller) [inline chef/data_bags/crowbar/template-sahara.json]: can we keep the upstream ordering or is there meaning in why we changed it?

https://github.com/openstack/sahara/blob/master/sahara/plugins/opts.py#L20

**Comment by toabctl** [comment]
> @Itxaka @toabctl mkcloud seems happy...

@vuntz Then I'm happy, too

File: chef/data_bags/crowbar/template-sahara.json (hunk 1 above)
Comment (vuntz) [inline chef/data_bags/crowbar/template-sahara.json]: The rationale is to have the items sorted alphabetically (except for the vailla one).

**Pull request merged**